Assets/_BForBoss/_Core/Scripts/Labor/RingSystem/RingLaborManager.cs

```diff
@@ -35,16 +32,24 @@ public void Initialize()
             _laborSystem = new LaborSystem(_ringSystems);
         }
 
+        public void ToggleTimer()
+        {
+            if(CountdownTimer.Instance.IsRunning)
```

Comment:
if you had to call this many `CountdownTimer.instance`, you can probably put this whole method intothe `countdownTimer`itself.

Reply:
Yeah good call, the toggle functionality will be a lot cleaner inside it as well